contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpSpanCustomizer.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http;
+
+import io.opencensus.contrib.http.util.HttpTraceUtil;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.SpanBuilder;
+import javax.annotation.Nullable;
+
+/**
+ * A helper class to customize an HTTP span.
+ *
+ * <p>It contains basic instrumentation implementation. See <a
+ * href="https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md">HTTP
+ * trace spec</a> for more information.
+ *
+ * <p>Users can extend this helper to customize builder and end status.
+ *
+ * @param <Q> the HTTP request entity.
+ * @param <P> the HTTP response entity.
+ * @since 0.13
+ */
+public class HttpSpanCustomizer<Q, P> {
```

Comment:
Can we make this package protected and avoid mentioning it in the README until we find an use-case where we really need it?

Reply:
Yes. Since we have a canonical way to instrument a span in the specification, we can use it as the default one and hide the customizer.

Reply:
Removed `public` modifier.